protos/model/data.proto

```diff
@@ -70,7 +70,10 @@ message ArtifactDataMap {
   map<string, ArtifactData> data = 1 [ json_name = "data" ];
 }
 
-message ArtifactData { int32 id = 1 [ json_name = "id" ]; }
+message ArtifactData { 
+  int32 id = 1 [ json_name = "id" ]; 
+  string text_map_id = 2 [ json_name = "text_map_id"];
+}
```

Comment:
Doesn't look like this matches the structure of how artifact data is currently generated.